### ISSUE TITLE
artists view - order albums chronologically

### ISSUE THREAD

**Issue by tbeitter** [Sonerezh/sonerezh#125]
Whilst alphanumerically sorting is fine for artists on "/artists", it would be really nice if the albums of a single artist could then be sorted by the year of release.

**Comment by lGuillaume124** [comment]
Absolutely, and we could imagine other filters too!

**Comment by nodiscc** [comment]
> Absolutely, and we could imagine other filters too!

Then I believe sorting songs by their file path would be an acceptable solution for https://github.com/Sonerezh/sonerezh/issues/28. Please consider reopening the issue if you think this makes sense.

**Comment by tbeitter** [comment]
@lGuillaume124 
Yes! I initially meant the sorting, but some filtering would be damn nice, too. First of all for genres!
